app/styles/app.css

```diff
@@ -46,3 +46,9 @@ body {
   width: 100%;
   padding: 16px;
 }
+
+.loading {
+  text-align: center;
+  margin-top: 5%;
+  font-size: 24px;
+}
```

Comment:
Can we have the loader in center of the screen?
Margin-top 5% really a bit off, creates a lot of empty space on rest of the page. After this change, we'll be good with merging the PR 🥳